docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,183 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: extend
+spec:
+  hosts:
+  - "*"
+  gateways:
+  - tlsgateway
+  tcp:
+  - match:
+    route:
+    - destination:
+        port:
+          number: 20000                    # the proxy server listening port
+        host: example-cluster-proxy-proxy  # the ClusterIP service name of the proxy server
+----
+
+Then a Coherence*Extend client can connect to the proxy server via TLS protocol at port 8043.  Below is an example of a <remoce-cache-scheme> configuration of a TLS Extend client for the above secure gateway example.  The server.jks in <trust-manager> must contain the same key/certificate as the one used by the gateway.
+
+client-cache-config.xml
+----
+...
+    <remote-cache-scheme>
+        <scheme-name>extend-direct</scheme-name>
+        <service-name>ExtendTcpProxyService</service-name>
+        <initiator-config>
+            <tcp-initiator>
+                <socket-provider>
+                    <ssl>
+                        <protocol>TLS</protocol>
+                        <trust-manager>
+                            <algorithm>PeerX509</algorithm>
+                            <key-store>
+                                <url>file:server.jks</url>
+                                <password>password</password>
+                            </key-store>
+                        </trust-manager>
+                    </ssl>
+                </socket-provider>
+                <remote-addresses>
+                    <socket-address>
+                        <address>127.0.0.1</address>
+                        <port>8043</port>
+                    </socket-address>
+                </remote-addresses>
+            </tcp-initiator>
+        </initiator-config>
+    </remote-cache-scheme>
+...
+----
+ 
+
+== Prometheus
+
+The coherence metrics that record and track the health of Coherence cluster using Prometheus are also available in Istio environment and can be viewed through Granfana.  However, Coherence cluster traffic is not visible by Istio.
+
+== Traffic Visualization
+
+Istio provides traffic management capabilities, including the ability to visualize traffic in Kiali. You do not need to change your applications to use this feature. The Istio proxy (envoy) sidecar that is injected into your pods provides it. The image below shows an example with traffic flow. In this example, you can see how the traffic flows in from the Istio gateway on the left, to the cluster services, and then to the individual cluster members.  This example has storage members (example-cluster-storage), a proxy member running proxy service (example-cluster-proxy), and a REST member running http server (example-cluster-rest).  However, Coherence cluster traffic between members is not visible.  
+
+image::../images/istioKiali.png[width=1024,height=512]
+
+To learn more, see https://istio.io/latest/docs/concepts/traffic-management/[Istio traffic management].
+
+
+== Limitations
```

Comment:
Move this to the beginning after the first paragraph of the Overview. These are important and should be at the beginning, otherwise people might not read all the way to the end.

Reply:
moved